ocaml-eglot-type-enclosing.el

```diff
@@ -134,7 +134,7 @@ If CURRENT is set, the range of the enclosing will be highlighted."
                       (font-lock-fontify-region (point-min) (point-max))
                       (buffer-string)))
     (display-buffer ocaml-eglot-type-buffer-name))
-  (when (and current (> (length ocaml-eglot-type-enclosing-types) 0))
+  (when (and current (not (null ocaml-eglot-type-enclosing-types)))
```

Comment:
Is this really doing the same thing ? Are lisp lists `null` when they are empty ?

Reply:
Yes, following my experiments. But I'll add some illustration test